packages/plugin-http-client/src/rest/mapper/Mapper.ts

```diff
@@ -0,0 +1,9 @@
+export class Mapper {
```

Comment:
Do we need to have a base class? I would go with an interface since this enables -> 
1) Abstract classes don't generally work well with TS and are usually pain to deal with
2) Interfaces allows for objects, the mappers don't need to be classes and since you usually only require these 2 methods